src/GitHub.InlineReviews/Services/PullRequestSession.cs

```diff
@@ -223,20 +223,27 @@ public async Task StartReview()
         }
 
         /// <inheritdoc/>
-        public async Task CancelReview()
+        public async Task<bool> CancelReview()
         {
             if (!HasPendingReview)
             {
                 throw new InvalidOperationException("There is no pending review to cancel.");
             }
 
-            await service.CancelPendingReview(LocalRepository, PendingReviewId);
+            if (service.ConfirmCancelPendingReview())
```

Comment:
Confirming the cancelation should be done in the view model, not the service. The view model should handle user interaction.